build/run-tests.sh

```diff
@@ -5,4 +5,4 @@ KUBECONFIG=${1-openshift.local.clusterup/openshift-apiserver/admin.kubeconfig}
 echo "Using KUBECONFIG '$KUBECONFIG'"
 
 go clean -testcache ./test/e2e
-go test -v ./test/e2e -timeout 45m
+go test -v ./test/e2e -run TestContainerCPUUpdateWithTwoReplicas -timeout 45m
```

Comment:
Isn't this used to run all the tests via Travis CI? This change would drop all but one this one change. (Also the file is named `run-tests`)

Reply:
@Crumby , yes, you are right, but it just for one issue reproducer. I will drop -run when this issue will be fixed, but I'm thinking about it nature.